### PR TITLE
Fix/ordering

### PR DESCRIPTION
**Pull request by illright**
This will adjust the ordering settings for the store and add ordering to the project browser.

### CODE REVIEW

File: innopoints/views/product.py [hunk 1]
```diff
@@ -32,10 +32,11 @@ def list_products():
...
     default_limit = 3
     default_page = 1
-    default_order = 'time'
+    default_order = 'name'
     ordering = {
-        'time': Product.addition_time,
-        'price': Product.price
+        'name': Product.name.asc(),
+        'price_asc': Product.price.asc(),
+        'price_desc': Product.price.desc()
```
Comment (aabounegm) [inline innopoints/views/product.py]: why did you remove ordering by time? It would help identify the new products.
and I would say ordering default to ascending, but why exclusively? I would suggest to rename the param to `order_by` and add `order=asc|desc` (actually, `order_by` as the param name makes more sense for the criteria even without specifying asc/desc)

File: innopoints/views/project.py [hunk 2]
```diff
@@ -32,7 +32,14 @@
...
     """List ongoing or past projects."""
-
+    first_activity = db.func.min(Activity.start_date)
+    default_order = 'creation_asc'
+    ordering = {
+        'creation_asc': Project.creation_time.asc(),
+        'creation_desc': Project.creation_time.desc(),
+        'proximity_asc': first_activity.asc(),
+        'proximity_desc': first_activity.desc(),
+    }
```
Comment (aabounegm) [inline innopoints/views/project.py]: *Same comment about `order_by` and setting asc/desc as another param*

**Review by aabounegm — APPROVED** [review]

File: innopoints/views/product.py [hunk 3]
```diff
@@ -32,25 +32,29 @@ def list_products():
...
         order = request.args.get('order', default_order)
     except ValueError:
         abort(400, {'message': 'Bad query parameters.'})
 
     if limit < 1 or page < 1:
         abort(400, {'message': 'Limit and page number must be positive.'})
 
-    if order not in ordering:
+    if (order_by, order) not in ordering:
+        print(order_by, order)
```
Comment (aabounegm) [inline innopoints/views/product.py:57]: `log.error`

Comment (illright) [reply innopoints/views/product.py:57]: Shit, no, this was a debug print)
